fitlins/workflows/base.py

```diff
@@ -196,15 +214,15 @@ def init_fitlins_wf(bids_dir, derivatives, out_dir, space, exclude_pattern=None,
                 name='{}_model'.format(level))
 
             wf.connect([
-                (models[-1], model, [('contrast_maps', 'stat_files')]),
-                (l1_metadata, model, [('out', 'stat_metadata')]),
-                (model, collate, [('contrast_metadata', 'contrast_metadata')]),
+                (stage, model, [('contrast_maps', 'stat_files'),
+                                ('contrast_metadata', 'stat_metadata')]),
```

Comment:
@adelavega I'm honestly not sure how `l1_metadata` worked here. It was always supposed to be this simple, and being able to drop the `l1_metadata` tag (thanks to bids-standard/pybids#319) allows us to do that.

Reply:
Hmm. yeah that makes way more sense to me... I'll try it out